### PR TITLE
Remove mptt

### PR DESCRIPTION
**Pull request by meshy**

### CODE REVIEW

File: conman/routes/tests/test_models.py [hunk 1]
```diff
@@ -295,6 +289,46 @@ def test_fall_back_to_branch(self):
...
+
+        self.assertEqual(descendants, [])
+
+    def test_descendants(self):
+        branch = ChildRouteFactory.create()
+
+        with self.assertNumQueries(1):
+            descendants = list(branch.parent.get_descendants())
+
+        self.assertEqual(descendants, [branch])
```
Comment (Ian-Foote) [inline conman/routes/tests/test_models.py]: You could avoid the `list` call with `assertSequenceEqual` I think.

Comment (meshy) [inline conman/routes/tests/test_models.py]: The list call is to nudge the query into happening within the context manager, not for the assert. I'd rather have the second assert outside it.

Comment (Ian-Foote) [inline conman/routes/tests/test_models.py]: Oh right! I always forget that's needed in these sorts of test.

Comment (meshy) [inline conman/routes/tests/test_models.py]: No worries `:)` I wish there was a more explicit way to tell the querysets into action -- this doesn't seen ideal.

**Comment by meshy** [comment]
@KevinEtienne @Ian-Foote can you see any reason not to merge this?

**Comment by Ian-Foote** [comment]
LGTM

**Comment by meshy** [comment]
:cake:

**Pull request merged**